### ISSUE TITLE
common-Accelerometer Calibration in Mission Planner

### ISSUE THREAD

**Issue by 1Steven** [diydrones/ardupilot-wiki-issue-tracker#300]
Unwisely, I upgraded to APM Plane 3.5 and MP 1.3.35 at the same time and now the accel calibration no longer works using the 3DR modem radio.  What's up?

**Comment by magicrub** [comment]
It should work, what's the problem?

**Comment by 1Steven** [comment]
Hello, 
When I initiate accel calibration for Plane, it never starts and will eventually time out. This morning, I retrograded to the prior APM Plane firmware and it works again. 
I found another issue too. I use battery mAh used in the HUD. In the latest MP stable release, it does not record mAh used unless I go to User Items, uncheck it and then check it again. My MinimOSD sees the mAh used consistently, so I'm pretty sure it's on the MP side.
Thanks,
Steven
